frontend/js/player-adapter-html5.js

```diff
@@ -96,7 +96,7 @@ define(
                          */
                         $(mediaElement).on("canplay durationchange", function () {
                             // If duration is still not valid
-                            if (isNaN(self.getDuration()) || mediaElement.readyState < 1) {
+                            if (!isFinite(self.getDuration()) || mediaElement.readyState < 1) {
```

Comment:
Wouldn't live streams have infinite durations? 🤔 Although to be fair, the rest of the tool can't deal with those, either. x)

Still, I wonder: Did you encounter a case where the duration is infinite first, and then changes to something finite?